nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java

```diff
@@ -536,7 +541,7 @@ public static PackageFormat getFormat(String textValue) {
                 return FLOWFILE_STREAM_FORMAT_V3;
             case "flowfile-stream-v2":
                 return FLOWFILE_STREAM_FORMAT_V2;
-            case "flowfile-stream-v1":
+            case "flowfile-tar-v1":
```

Comment:
I can't help but hope there is a better way to avoid duplicating these strings in the code, but it's fine for now.